AboutMe.Web/Program.cs

```diff
@@ -0,0 +1,49 @@
+using AboutMe.HelperExtensions;
+using Infrastructure;
+using Application.AppStart;
+using Core.Entities;
+using Microsoft.AspNetCore.Identity;
+using Core.Mapping;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddRazorPages();
+builder.Services.AddContext(builder.Configuration.GetConnectionString("DefaultConnection"));
+builder.Services.AddServices();
+builder.Services.AddAutoMapper(typeof(AutoMapperProfile));
+
+builder.Services.AddIdentity<User, Role>(options =>
+{
+    options.Password.RequireDigit = false;
+    options.Password.RequireLowercase = false;
+    options.Password.RequireNonAlphanumeric = false;
+    options.Password.RequireUppercase = false;
+}).AddEntityFrameworkStores<AppDbContext>().AddDefaultTokenProviders();
+
+
+var app = builder.Build();
+
+app.SetupDataBase();
+
+if (!app.Environment.IsDevelopment())
```

Comment:
I would suggest to have to separate pipelines for dev and prod, that will be easier to maintain

Reply:
I dont understand your proposal
i have custom error handling for production evironment
if i will need to add a condition for dev environment, i will add "else"

Reply:
If your project will grow you will have to make your Program.cs bigger and more complicated. To modify it you will have to read through it all and understand that initialization flow. If your Program.cs will grow to 400+ line it will be a  nightmare.